stacks/container-apps/augury-alarms.yml

```diff
@@ -0,0 +1,46 @@
+# stacks/container-apps/augury-alarms.yml
+AWSTemplateFormatVersion: "2010-09-09"
+Description: Augury alarms
+Parameters:
+  OpsWarnMessagesSnsTopicArn:
+    Type: String
+  OpsErrorMessagesSnsTopicArn:
+    Type: String
+  OpsFatalMessagesSnsTopicArn:
+    Type: String
+  EnvironmentType:
+    Type: String
+  EnvironmentTypeAbbreviation:
+    Type: String
+  AuguryWorkerLogGroupName:
+    Type: String
+Resources:
+  AuguryActualsIngestJobInvocationMetric:
+    Type: "AWS::Logs::MetricFilter"
+    Properties:
+      FilterPattern: '[ImpressionsFetchJob] Success'
+      LogGroupName: !Ref AuguryWorkerLogGroupName
+      MetricTransformations:
+        - MetricName: !Sub "augury_actuals_rollups_ct_${EnvironmentTypeAbbreviation}"
+          MetricNamespace: LogMetrics
+          MetricValue: "1"
+  AuguryActualsAreBehindAlarm:
+    Type: "AWS::CloudWatch::Alarm"
+    Properties:
+      ActionsEnabled: true
+      AlarmName: !Sub "[Augury][Worker][ActiveJob] ${EnvironmentType} Augury actuals are behind"
+      AlarmActions:
+        - !Ref OpsErrorMessagesSnsTopicArn
+      InsufficientDataActions:
+        - !Ref OpsErrorMessagesSnsTopicArn
+      OKActions:
+        - !Ref OpsErrorMessagesSnsTopicArn
+      AlarmDescription: Augury actuals are behind
+      ComparisonOperator: LessThanThreshold
+      EvaluationPeriods: 3
+      MetricName: AuguryActualsIngestJobInvocationMetric
+      Namespace: AWS/ECS
+      Period: 300
+      Statistic: Sum
+      Threshold: 1
```

Comment:
I'm reading this as: there has to be at least one invocation in 15 minutes or the alarm goes off.

Reply:
Ahh, right.  So I think 5 minutes is fine here, just was incorporating the `EvaluationPeriods` in that overall view.

Rephrasing I imagine this to work: if in 3 consecutive periods of 5 minutes each, if there are no matching log lines (indicating a successful actuals-ingest job completion) then the alarm goes off.